### ISSUE TITLE
Duplicate parameters when POST with parameters

### ISSUE THREAD

**Issue by rubenjgarcia** [dispatch/reboot#143]
I have this code

```
var req = url("http://localhost:8000")
req = req <<? Map("a" -> "b") // Here url.req is http://localhost:8000?a=b
req = req << """{}""" setContentType("application/json", "UTF-8") // Here the url is http://localhost:8000?a=b&a=b
```

**Comment by fbascheper** [comment]
Thanks for reporting this, however it would help greatly if you could at least describe the version affected. Please provide a working testcase against the latest version, then we can have a look at this.

**Comment by rubenjgarcia** [comment]
I'm using version 0.12.0

You can reproduce the issue with this code

```
import dispatch._

object Test extends App {

  var req = url("http://localhost:8000")
  req = req <<? Map("a" -> "b")
  println(req.url)
  req = req << """{}""" setContentType("application/json", "UTF-8")
  println(req.url)
}
```

The output is

```
http://localhost:8000?a=b
http://localhost:8000?a=b&a=b
```

**Comment by ashawley** [comment]
Sounds a lot like #138.  I don't think it made the 0.12.0 release.  I think we can mark this as a dup, for now, and close once it's cherry-picked to 0.12 and released with 0.12.1.

**Comment by farmdawgnation** [comment]
This should be resolved in 0.12.1. Please open a new issue if you find it isn't.
